lib/models/villager_model.dart

```diff
@@ -0,0 +1,42 @@
+class villager {
+  String? name;
+  String? imageUrl;
+  String? species;
+  String? personality;
+  String? gender;
+  String? birthdayMonth;
+  String? birthdayDay;
+
+  villager({
+    this.name,
+    this.imageUrl,
+    this.species,
+    this.personality,
+    this.gender,
+    this.birthdayMonth,
+    this.birthdayDay,
+  });
```

Comment:
if everything is supposed to be optional and since there aren't any default values, what happens if the values come as null?